test/e2e-common.sh

```diff
@@ -515,3 +517,7 @@ function setup_kafka_channel_auth() {
       -p='[{"op": "remove", "path": "/data/auth.secret.ref.name"}, {"op": "remove", "path": "/data/auth.secret.ref.namespace"}]' || true
   fi
 }
+
+function install_eventing_core_test_tls_resources() {
+    ko apply -Rf "${repo_root_dir}/vendor/knative.dev/eventing/test/config/tls" || return $?
```

Comment:
```
Error from server (NotFound): error when creating "STDIN": namespaces "knative-eventing" not found
Error: error executing 'kubectl apply': exit status 1
```
Seems we need to create the namespace before

Reply:
Placed the call after installing Eventing core